twitcher/magpieconfig.py

```diff
@@ -0,0 +1,20 @@
+from pyramid.authentication import AuthTktAuthenticationPolicy
+from pyramid.authorization import ACLAuthorizationPolicy
+import os
+from magpie.models import group_finder
```

Comment:
This is changed in Magpie 0.6.x (WIP)
https://github.com/Ouranosinc/Magpie/blob/cba7e014e24861233f1e9b6f0ebe477fe9c284d1/magpie/definitions/ziggurat_definitions.py#L3

I can change the line to `from ziggurat_foundations.models import groupfinder `**`as group_finder`** or add some definition like `group_finder = groupfinder` in `models.py` to ensure compatibility but will have to test whether the function is still found since it's not directly referenced (should be imported via `models.py` importing `ziggurat_definitions.py`)

Reply:
If groupfinder is a function from ziggurat we should indeed import it from the original package not from another package importing it. If group_finder from magpie.models was only an alias over groupfinder from zig we should only rename that function in twitcher/magpieconfig.py